go/kbfs/libfuse/dir.go

```diff
@@ -285,7 +285,7 @@ func (f *Folder) batchChangesInvalidate(ctx context.Context,
 			}
 			for _, name := range v.DirUpdated {
 				// invalidate the dentry cache
-				if err := f.fs.fuse.InvalidateEntry(n, name); err != nil && err != fuse.ErrNotCached {
+				if err := f.fs.fuse.InvalidateEntry(n, name.Plaintext()); err != nil && err != fuse.ErrNotCached {
 					// TODO we have no mechanism to do anything about this
 					f.fs.log.CErrorf(ctx, "FUSE invalidate error: %v", err)
```

Comment:
Good question. Checking the bazil code, it seems like this would only return syscall-style error codes, nothing with actual plaintext names.  Definitely seems like a good thing to guard against in general and we should keep an eye out for whether it's needed, but I think for now I'll probably punt.  Seems like a big-ish job to search-and-replace every error, and without knowing for sure it's needed, it's probably not worth it right now.  I could be persuaded though.

Reply:
I think your approach sounds reasonable to me.